packages/svelte-materialify/@types/index.d.ts

```diff
@@ -42,6 +42,9 @@ export { default as Slider } from './Slider';
 export { default as Snackbar } from './Snackbar';
 export { default as Subheader } from './Subheader';
 export { default as Switch } from './Switch';
+export { default as Tab } from './Tab';
+export { default as Tabs } from './Tabs';
+export { default as TabsContent } from './TabsContent';
 export { default as Textarea } from './Textarea';
 export { default as TextField } from './TextField';
-export { default as Tooltip } from './Tooltip';
+export { default as Tooltip } from './Tooltip';
```

Comment:
I think you have some sort of formatting which kills our formatting (no expert on this). Files should have a newline at the end, easiest is probably to run `yarn lint:fix` on the root folder of the repo 👍